packages/zowe-explorer/src/uss/actions.ts

```diff
@@ -82,17 +77,13 @@ export async function createUSSNode(
                 vscode.window.showInformationMessage(
                     localize(
                         "createUSSNode.name.exists",
-                        "There is already a file with same name. Please change your OS file system settings if you want to give case sensitive file names."
+                        "There is already a file with the same name. Please change your OS file system settings if you want to give case sensitive file names."
```

Comment:
Same question as above - should we split across multiple lines to make the linter happy?

Reply:
See above comment regarding lint warnings